can_tools/scrapers/cache_scraper_versions.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
not sure if this is the best location or format for this file

Reply:
Good points. 

For question 2, I restructured this and now each scraper creates it's own file (and executes the scraper if the file does not exist so that these files don't need to be checked in.)

You make a good point and I agree that it doesn't feel great to be modifying files without any commit history :/